go/base/management/commands/go_manage_routing_table.py

```diff
@@ -51,48 +43,46 @@ def handle(self, *args, **options):
                 else:
                     raise CommandError('Please provide %s:' % (opt.dest,))
 
-        command_options = [c for c in self.CONFLICTING_OPTIONS if options[c]]
-        if len(command_options) != 1:
+        # Ensure a single option is chosen from the local options
+        cmd_options = [c.dest for c in self.LOCAL_OPTIONS if options[c.dest]]
+        if len(cmd_options) != 1:
             raise CommandError('Please provide exactly one of: %s' % (
-                ['--%s' % c for c in self.CONFLICTING_OPTIONS],))
-
-        user = get_user_by_email(options['email-address'])
-        user_api = vumi_api_for_user(user)
+                ['--%s' % c.dest for c in self.LOCAL_OPTIONS],))
```

Comment:
Using `LOCAL_OPTIONS` here is a little less clear than the old `CONFLICTING_OPTIONS` mechanism, even with the explanatory comment. It's also problematic if we add a new non-conflicting option later. Any thoughts about a way to improve this?